src/runtime/start.ts

```diff
@@ -114,3 +103,11 @@ export function createStartModuleContent(config: StartModuleConfig): string {
 
   return output
 }
+
+export function prepareStartModule(
+  tsBuilder: EmitAndSemanticDiagnosticsBuilderProgram,
+  startModule: string
+): string {
+  log.trace('Transpiling start module')
+  return transpileModule(startModule, tsBuilder.getCompilerOptions())
+}
```

Comment:
When in dev mode, we use the TS instance provided by ts-node to transpile, not this one. Unifying this would be nice one day. I find it confusing.

Reply:
The way I see it, `ts-node` should only be used for it's "on-the-fly" ts compilation. If you want to be consistent here, I'd suggest to use our internal `transpileModule` function in `tsc.ts`

Reply:
@Weakky That's what I had before, but it added a whopping 1000ms latency because a new builder had to be created. I didn't actually try passing the ts-node instance to it this time, but IIRC, it won't work.